scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -207,8 +207,8 @@ def cancel_trailing_jobs(start_time: datetime) -> bool:
 
     print(
         f"⚠️ Cancelling {len(jobs)} job(s) created after {start_time}.\n"
-        "Add any notebooks that submit jobs to NOTEBOOKS_EXCLUDE in "
-        "`scripts/nb-tester/test-notebook.py`."
+        "Add any notebooks that submit jobs to `notebooks-that-submit-jobs` in "
```

Comment:
Style nit: consider having the arg be `config_path: str`, then the caller passes `arg.config_path` directly. It keeps the scope of this function smaller and easier to anticipate in `_main()` what the function does.